temba/channels/types/firebase/type.py

```diff
@@ -56,6 +57,12 @@ def send(self, channel, msg, text):
             }
             data['content_available'] = True
 
+        metadata = msg.metadata if hasattr(msg, 'metadata') else {}
+        quick_replies = metadata.get('quick_replies', [])
+
+        if quick_replies:
+            data['data']['metadata'] = quick_replies
+
```

Comment:
Few questions here:

1) Why is this in firebase at all? Not sure I understand how that would surface on the Android Channel which is what firebase is used for.

2) Why is this stuffing quick_replies in something called metadata? Is there a spec somewhere you can share? Feels weird that we get quick_replies out of something we call metadata and then assign it to somebody else's metadata.

Reply:
I agree about the "metadata" nomenclature, it could be `quick_replies`, right? But about the data of quick replies, this can be handled on the mobile app side, that allows the developer implement the quick replies buttons like other platforms.